### DISCUSSION TITLE
No sign-in page after fresh install

### DISCUSSION THREAD

**Discussion by cbirkenbeul** [teslamate-org/teslamate#1278] [Q&A]
I installed teslamate with own manifests in kubernetes. The admin Dashboard works fine, beside of the possibility of login to my tesla account. The homepage always jump to the TeslaFi import screen.

There was a older issue with the same behavior, but without any solution.

Tired it some weeks ago with the 1.21 version. with 1.21.2 still the same issue.

Logs does not provide any usefull information. Just 302 to /import messages.

**Comment by adriankumpf** [comment]
The solution is described on the documentation page for the [TeslaFi import](https://docs.teslamate.org/docs/import/teslafi/):

> 4. After the import is complete, **empty the `import` directory** (or remove but ensure docker doesn't have a volume mapping) and **restart** the `teslamate` service.

**Reply by cbirkenbeul** [reply]
But i dont want to import because i dont have any teslafi data. 

I tried teslamate some month ago and it worked without import.

**Reply by adriankumpf** [reply]
Oh, okay, I assumed that you already did an import. In that case, re-check your Kubernetes configuration and have a look which directory you have mounted at `/opt/app/import`. If TeslaMate finds an existing directory at startup, and there are files in it, the import screen will be displayed.
